lib/app/modules/Settings/views/settings_view.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:managment_app/Themes/themes.dart';
+import 'package:managment_app/components/SettingButton.dart';
+import '../../../../components/Logoutbutton.dart';
+import '../controllers/settings_controller.dart';
+
+class SettingsView extends GetView<SettingsController> {
+  const SettingsView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return Scaffold(
+      body: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              gradient: LinearGradient(
+                colors: [theme.colorLevel1, theme.colorLevel2],
+                begin: Alignment.topCenter,
+                end: Alignment.bottomCenter,
+              ),
+            ),
+          ),
+          SingleChildScrollView(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
```

Comment:
give mainAxis alignment and make the page responsive such that no need to scroll this page